ghost/admin/app/authenticators/cookie.js

```diff
@@ -33,6 +33,9 @@ export default Authenticator.extend({
         }
 
         const data = {username: identification, password};
+        if(key) {
```

Comment:
With the above, if it was `skipEmailVerification`, this could be:

```
if (skipEmailVerification) {
    data.skipEmailVerification = true;
}
```